src/components/SearchBox/SearchBox.scss

```diff
@@ -0,0 +1,50 @@
+.searchbox {
+    $input-font-size: 30px;
```

Comment:
I think it's better to use em's so we can handle responsive changes easily in the future

Reply:
Also, I think we should move our scss variables out into a separate file so that they can be imported anywhere  and we can change them in one place if need be.

Reply:
These variables are local to this file. The variables defined here are just intended for this file.

Reply:
For the em's. There is a separate bug that will cover px -> em's. #29